src/cocotb/types/__init__.py

```diff
@@ -4,7 +4,22 @@
 from ._abstract_array import AbstractArray
 from ._array import Array
 from ._logic import Logic
-from ._logic_array import LogicArray
+from ._logic_array import RESOLVE_X, LogicArray, ResolveX
```

Comment:
You can't import `RESOLVE_X` here, it's intended to be a global you set. But that's going to be refactored by #4622.

Reply:
Fixed!